experimental/IntersectionTheory/src/Bott.jl

```diff
@@ -80,10 +80,91 @@ The type of a abstract_variety with a torus action, represented by the fixed poi
   end
 end
 
-euler(X::TnVariety) = sum(1//ZZ(e) for (p,e) in X.points) # special case of Bott's formula
+@doc raw"""
+    tn_variety(n::Int, points::Vector{Pair{P, Int}}) where P
+
+Return an abstract_variety with a torus action, represented by the fixed points.
```

Comment:
In which way is the variety represented by its fixed points? 
It might be useful to give an explanation or a reference.

Reply:
This PR focuses on functionality for further discussions. A subsequent PR will bring a thorough introduction to varieties with a torus action and Bott's formula, including what you requested above (please keep in mind that we work with abstract varieties here: The results apply to all smooth projective varieties with the given data (here, admitting a torus action with the given number of fixed points and multiplicities).